worlds/uuv_hippocampus.world

```diff
@@ -27,13 +27,9 @@
         Include the model of the Hippocampus from TUHH.
         The pose is chosen that the hippocampus starts at position (0,0,0) with its own coordinate frame equal to the
         NED World frame.
+        Above is currently broken.
```

Comment:
I think we can remove that comment altogether

Reply:
Done!